dm/dm-precheck.md

```diff
@@ -94,9 +99,9 @@ For the incremental data migration mode (`task-mode: incremental`), in addition
     - REPLICATION CLIENT permission
     - REPLICATION SLAVE permission
 
-* (Mandatory) Database primary-secondary configuration
+* Database primary-secondary configuration
 
-    - The database ID `server_id` of the upstream database must be specified (GTID is recommended for non-AWS Aurora environments).
+    - It is recommended that you specify the database ID `server_id` for the upstream database (GTID is recommended for non-AWS Aurora environments), so as to avoid primary-secondary replication failures.
```

Comment:
```suggestion
    - To avoid primary-secondary replication failures, it is recommended that you specify the database ID `server_id` for the upstream database (GTID is recommended for non-AWS Aurora environments).
```